### ISSUE TITLE
populate does not work like create

### ISSUE THREAD

**Issue by nicolasfranck** [glenrobson/SimpleAnnotationServer#18]
I tried using the controller "/populate" (POST). It seemed to work, but when
trying to search on uri ("/annotation/search") it return an empty array.

When I created the annotations directly by posting to "/annotation/create",
it worked. Any idea why?

**Comment by glenrobson** [comment]
it should work the same as /create. Is the annotation list your posting public at all? If so I can try running it against my copy to see if I can find the issue. Are you seeing anything in the console when you run the populate post?

Cheers

Glen

**Comment by nicolasfranck** [comment]
try this one:

  http://adore.ugent.be/local/annotationlist.json

**Comment by glenrobson** [comment]
Found the issue. The annotationlist I've been testing with doesn't have an @id for the annotation but yours does (which is correct). Part of my code added a @context to the annotation so it could be converted to RDF but it incorrectly only did this if there wasn't an id. Fixed this now so it adds it irrespective of if it has an @id or not.

I've committed a fix but let me know if it doesn't work for you.

Cheers

Glen
